pgjdbc/src/main/java/org/postgresql/jdbc/PgCallableStatement.java

```diff
@@ -657,11 +658,11 @@ public String getNString(@Positive int parameterIndex) throws SQLException {
 
   public <T> @Nullable T getObject(@Positive int parameterIndex, Class<T> type)
       throws SQLException {
-    if (type == ResultSet.class) {
-      return type.cast(getObject(parameterIndex));
+    if (type == ResultSetMetaData.class) {
+      throw new PSQLException(GT.tr("Unsupported type conversion to {1}.", type),
```

Comment:
It seems to me the {1} substitution does not work as you can see in stack trace: 
```
org.postgresql.util.PSQLException: Unsupported type conversion to {1}.
	at org.postgresql.jdbc.PgCallableStatement.getObject(PgCallableStatement.java:663)
```

Reply:
ah, good catch, thanks, but the rest works, right ?

Reply:
@anatoly-spb ??

Reply:
It looks good